src/js/profile/wearable/widget/wearable/ArcListview.js

```diff
@@ -843,6 +898,62 @@
 				}
 			};
 
+			/**
+			 * Add new item to listview
+			 * @method addItem
+			 * @param {HTMLElement} liElement new list item
+			 * @param {number} index item index on list, default last item
+			 * @memberof ns.widget.wearable.ArcListview
+			 */
+			prototype.addItem = function (content, index, liElement) {
+				var li = liElement || document.createElement("li"),
+					lastItem,
+					prevItem;
+
+				// append new li elements to widget element
+				if (typeof this.options.listItemUpdater === "function") {
+					this.options.listItemUpdater(li, index);
+				} else {
+					li.innerHTML = "<a href=\"\">" + content + "</a>";
+				}
+				this.element.appendChild(li);
+
+				// find new li elements attached to widget element
+				this._addItemsFromElement();
+				// move li elements to widget cache
+				this._setAnimatedItems();
+
+				// set new item position on list;
+				if (this._state.items.length > 1) {
+					lastItem = this._state.items[this._state.items.length - 1];
+					prevItem = this._state.items[this._state.items.length - 2];
+
+					lastItem.y = prevItem.y + prevItem.height;
+					lastItem.rect.bottom = prevItem.rect.bottom + prevItem.rect.height;
+					lastItem.rect.top = prevItem.rect.top + prevItem.rect.height;
+				}
+
+				this._setMaxScrollY();
+				this._bouncingEffect._maxValue = this._maxScrollY;
+
+				// refresh widget view
+				this.refresh();
+			}
+
+			/**
+			 * Remove item at indicated index
+			 * @method removeItem
+			 * @param {number} index item index on list, default last item
```

Comment:
Implementation does not allow to use method without passing index

Reply:
All this method has removed.